### PR TITLE
Dev 1402 fix UI/UX issue subject & icons are superimpose

### PR DESCRIPTION
**Pull request by ahsinsiddique**

### CODE REVIEW

File: src/app/mail/mail-detail/mail-detail.component.scss [hunk 1]
```diff
@@ -124,3 +124,13 @@
   right: 30px;
   top: 23px;
 }
+
+.mail-subject {
+  max-width: 97%
+}
+
+@media (max-width: 576px) {
```
Comment (atifsaddique211f) [inline src/app/mail/mail-detail/mail-detail.component.scss]: use bootstrap media query `<=sm`

Comment (ahsinsiddique) [reply src/app/mail/mail-detail/mail-detail.component.scss]: Okay sure

File: src/app/mail/mail-detail/mail-detail.component.scss [hunk 2]
```diff
@@ -1,3 +1,7 @@
+@import '../../../styles/vendors/include-media/include-media';
+@import '../../../styles/base/variables';
+@import '../../../styles/config/include-media-config';
+@import '../../../styles/vendors/ceaser/ceaser-easing';
```
Comment (atifsaddique211f) [inline src/app/mail/mail-detail/mail-detail.component.scss]: remove unused imports

Comment (ahsinsiddique) [reply src/app/mail/mail-detail/mail-detail.component.scss]: removed

**Pull request merged**